examples/basic/main.rs

```diff
@@ -1,17 +1,15 @@
 #[macro_use]
 extern crate tera;
-#[macro_use]
-extern crate lazy_static;
 extern crate serde_json;
 
 use std::collections::HashMap;
 
-use tera::{Tera, Context, Result};
+use tera::{Tera, Context, make_filter, Result};
 use serde_json::value::{Value, to_value};
 
 
-lazy_static! {
-    pub static ref TEMPLATES: Tera = {
+thread_local! {
```

Comment:
Not being `lazy_static` anymore makes me sad :(
It complicates/slows down the use for 99% of Tera users, I am guessing using `Arc` is a bad idea?

I'll play with this branch on Gutenberg to see the impact

Reply:
I mean since we take `'static`, shouldn't be with be able to use `Arc` instead of `Rc` to make it `Sync` @andy128k ?

Reply:
Done